packages/api/src/feedback.ts

```diff
@@ -24,3 +24,11 @@ export interface FeedbackProperties {
   comment?: string;
   contact?: string;
 }
+
+export interface GitHubIssueProperties {
+  category: FeedbackCategory;
+  issueTitle: string;
+  issueDescription: string;
```

Comment:
```suggestion
export interface GitHubIssue {
  category: FeedbackCategory;
  title: string;
  description: string;
```

When can simplify the interface, we don't need to use `Properties` suffix to the interface, nor `issue` prefix